### PR TITLE
Make the gem github include code more concise

### PR DESCRIPTION
**Pull request by kieranklaassen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ gem 'bootstrap-wysihtml5-rails'
 or you can install from latest build;
 
 ``` ruby
-gem 'bootstrap-wysihtml5-rails', :git => 'git://github.com/Nerian/bootstrap-wysihtml5-rails.git'
+gem 'bootstrap-wysihtml5-rails', github: 'nerian/bootstrap-wysihtml5-rails'
```
Comment (Nerian) [inline README.md]: The address is 'Nerian'. Does it make a difference 'nerian' vs 'Nerian'?

Comment (kieranklaassen) [inline README.md]: There is no functional difference. I prefer links to be lowercase. It’s up to you to decide what you want.

Comment (Nerian) [inline README.md]: If the repo address works in lowercase all is good :)

**Comment by Nerian** [comment]
Thanks

**Pull request merged**